### PR TITLE
EX-1687 graphql commands

### PR DESCRIPTION
**Pull request by nlinker**
Implements GraphQL for commands.
Also fixes some clippy warnings.

Screenshot 2020-09-14 at 18 41 25
image

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2240)

### CODE REVIEW

**Comment by ip1981** [comment]
We usually need a command with specific id, no?

**Comment by nlinker** [comment]
> We usually need a command with specific id, no?

image
The original task however haven't mentioned this.

File: iml-api/src/graphql.rs [hunk 1]
```diff
@@ -260,6 +261,77 @@ impl QueryRoot {
...
+                        c.id AS id,
+                        cancelled,
+                        complete,
+                        errored,
+                        created_at,
+                        array_agg(cj.job_id)::integer[] AS job_ids,
+                        message
+                    FROM chroma_core_command c
+                    JOIN chroma_core_command_jobs cj ON c.id = cj.command_id
+                    WHERE complete = $4 AND ($5::TEXT IS NULL OR c.message LIKE '%' || $5 || '%')
```
Comment (ip1981) [inline iml-api/src/graphql.rs:295]: Maybe ILIKE?

Comment (nlinker) [reply iml-api/src/graphql.rs:295]: Agree, good idea.

**Review by mkpankov — APPROVED** [review]
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2240)* status: all files reviewed, 1 unresolved discussion (waiting on @ip1981)

**Review by mkpankov — APPROVED** [review]
> Reviewed 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2240)* status: all files reviewed, 1 unresolved discussion (waiting on @ip1981)

**Review by ip1981 — DISMISSED** [review]

**Comment by jgrund** [comment]
Looks like sqlx schema check is failing, otherwise LGTM.

**Review by ip1981 — APPROVED** [review]

**Review by jgrund — APPROVED** [review]

**Pull request merged**